root/static/scripts/common/coverart.js

```diff
@@ -11,24 +11,23 @@ import $ from 'jquery';
 $(function () {
     $(".cover-art-image").each(function () {
         var $e = $(this);
-        var thumbnail_url = window.devicePixelRatio > 1
+        var thumbnailUrl = window.devicePixelRatio > 1
             ? $e.data("large-thumbnail")
             : $e.data("small-thumbnail");
 
         $("<img />")
             .bind('error', function () {
-                if ($e.data("fallback") && $e.attr("src") === thumbnail_url) {
+                if ($e.data("fallback") && $e.attr("src") === thumbnailUrl) {
                     $e.attr("src", $e.data("fallback"));
                 } else {
                     $e.closest('a')
                         .replaceWith('<em>' + $e.data("message") + '</em>');
                 }
-            })
-            .attr({
-                "src": thumbnail_url,
-                "title": $e.data("title")
-            })
-            .appendTo(this);
-
+        })
+        .attr({
+            "src": thumbnailUrl,
+            "title": $e.data("title")
+        })
+        .appendTo(this);
```

Comment:
The rest of the PR looks good, but the indentation here got changed (possibly while resolving conflicts) - can you fix that when you have time? :)